src/main/java/core/basesyntax/Parameter.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Parameter {
```

Comment:
It is not a good name for this interface. "Parameter" does not explain what this interface do.
Better use name: "AreaCalculator"